google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/AbstractMockServerTest.java

```diff
@@ -328,6 +328,14 @@ boolean isMultiplexedSessionsEnabled(Spanner spanner) {
     return spanner.getOptions().getSessionPoolOptions().getUseMultiplexedSession();
   }
 
+  boolean isMultiplexedSessionsEnabledForPartitionedOps(Spanner spanner) {
+    if (spanner.getOptions() == null || spanner.getOptions().getSessionPoolOptions() == null) {
+      return false;
+    }
+    return spanner.getOptions().getSessionPoolOptions().getUseMultiplexedSessionPartitionedOps()
```

Comment:
getUseMultiplexedSessionPartitionedOps() should already have been checking for getUseMultiplexedSession()?
So we can remove the other condition?

Reply:
Fixed.